go.mod

```diff
@@ -62,3 +62,5 @@ require (
 	google.golang.org/grpc v1.48.0 // indirect
 	google.golang.org/protobuf v1.28.1 // indirect
 )
+
+replace github.com/microsoft/azure-devops-go-api/azuredevops => github.com/josh-barker/azure-devops-go-api/azuredevops v1.0.0-b5-checkconfig
```

Comment:
Suggest fix the SDK issue like [pipeline check](https://github.com/microsoft/terraform-provider-azuredevops/blob/main/azuredevops/internal/utils/pipelineschecksextras/pipelineschecks_extras.go) does, patch in this project not in another personal SDK.